examples/seismic/model.py

```diff
@@ -215,6 +214,13 @@ def __init__(self, origin, spacing, shape, space_order, vp, nbl=20,
         if self.grid.dim > 2:
             self.phi = self._gen_phys_param(phi, 'phi', space_order)
 
+    @property
+    def _max_vp(self):
+        # mmax not supported with yask
+        if configuration['backend'] == 'yask':
```

Comment:
this `if` cannot be here

how does it fail with yask? is there anything we can do about that?

shouldn't this check go inside `mmax` / `mmin` ?

Reply:
no `data_ro_domain` error with task. Basically had to turn `)max_vp` in a property because pickling the model was messing with it with the previous version. I'll move it to max